addons/sourcemod/configs/scp_sf/weapons.cfg

```diff
@@ -928,7 +928,7 @@
 	"30020"	// Combat Armor
 	{
 		"classname"	"tf_weapon_fists"
-		"attributes"	"62 ; 0.68 ; 64 ; 0.74 ; 66 ; 0.74 ; 252 ; 0.6 ; 476 ; 0"
+		"attributes"	"54 ; 0.95 ; 62 ; 0.9 ; 64 ; 0.7 ; 66 ; 0.7 ; 252 ; 0.5 "
```

Comment:
Attrib 62 change would've made user take more damage (0.68 to 0.9)

Reply:
Intentional to keep headshots deadly, but might lower it to 0.75